src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -10,14 +10,16 @@ public class PointerEventArgs : EventArgs
 	{
 
 		Func<IElement?, Point?>? _getPosition;
+		public PlatformPointerEventArgs? PlatformArgs { get; private set; }
```

Comment:
Missing docs! 

Bonus points if you also took the time to document some of the public symbols here :)

Reply:
Let me know if these are okay documentation!